test/presubmit-tests.sh

```diff
@@ -119,6 +132,12 @@ run_unit_tests() {
   (
     set -eu -o pipefail
 
+    cd $CLI_DIR
+    cli-build
```

Comment:
Any reason for not having `cli-build` in `run_build_tests` ?

Reply:
We should revise the whole of the CI script in a different PR. This just follows the pattern used for both api and ui.

Reply:
Although its outside the scope of this PR, I thought we should fix this for once and for all, hence the last commit that fixes it up. 
Thanks a lot for pointing this out @vdemeester .